front-end/my-vite-app/src/Survey.jsx

```diff
@@ -1,24 +1,28 @@
-import { useState, useContext } from 'react';
+import { useState } from 'react';
 import './Survey.css';
 import { useNavigate } from 'react-router-dom';
 import { UserContext } from './UserContext.js';
 
 const Survey = ({closeModal}) => {
-    const [step, setStep] = useState(1)
-    const [interest, setInterest] = useState('')
-    const [level, setLevel] = useState('')
-    const [rating, setRating] = useState('')
-    const [languages, setLanguages] = useState('')
-    const [courses, setCourses] = useState([])
-
-    const nextStep = () => setStep(step+1)
-    const prevStep = () => setStep(step-1)
+    const [step, setStep] = useState(1);
+    const [interest, setInterest] = useState('');
+    const [level, setLevel] = useState('');
+    const [rating, setRating] = useState('');
+    const [languages, setLanguages] = useState([]);
+    const [courses, setCourses] = useState([]);
+    const [recommendedCourses,  setRecommendedCourses] = useState([]);
+    const [className, setClassName] = useState('light-button')
 
+    const nextStep = () => setStep(step + 1);
+    const prevStep = () => setStep(step - 1);
```

Comment:
Great, you can use this to also implement a visualization for the survey progress bar?